pkg/libvirttools/virtualization.go

```diff
@@ -277,6 +280,58 @@ func (v *VirtualizationTool) addVolumesToDomain(podID, podName string, domain *l
 	return nil
 }
 
+func vmLogToFile() bool {
+	if os.Getenv("VIRTLET_VM_LOG_TO_FILE") != "" {
+		return true
+	}
+	return false
+}
+
+func (v *VirtualizationTool) RemoveLibvirtSandboxLog(sandboxId string) error {
+	return os.RemoveAll(filepath.Join(vmLogBaseDir, sandboxId))
+}
+
+func (v *VirtualizationTool) addSerialDevicesToDomain(sandboxId string, domain *libvirtxml.Domain, settings VirtletDomainSettings) error {
+	if settings.vmLogToFile {
+		logDir := filepath.Join(vmLogBaseDir, sandboxId)
```

Comment:
Do you need a separate dir? Wouldn't it be sufficient if the log would be created as `vmLogBaseDir/sandboxId_raw.log`